specification/maps/data-plane/readme.md

```diff
@@ -42,22 +42,40 @@ directive:
     from: search.json
     where: $.paths["/providers/Microsoft.Subscription/subscriptionOperations/{operationId}"].get
     reason: The reason for this suppression is the API is already released and introducing new LRO properties will not function and are not supported today and will only be developed for the next version of this API.
-``` 
 
+  - suppress: INVALID_TYPE
+    reason: false positive from oav is breaking our example validation. See azure/oav#1020.
+
+  - suppress: RESPONSE_SCHEMA_NOT_IN_SPEC
+    reason: false positive from oav is breaking our example validation. See azure/oav#1021.
+```
+
+### Tag: package-stable-2023-07-01
+
+These settings apply only when `--tag=package-stable-2023-07-01` is specified on the command line.
+
+```yaml $(tag) == 'package-stable-2023-07-01'
+input-file:
+  - Common/stable/2023-06-01/common.json
+  - Microsoft.Maps/Data/preview/2.0/data.json
+  - DataRegistry/stable/2023-06-01/dataregistry.json
+  - Geolocation/preview/1.0/geolocation.json
+  - Render/stable/2022-08-01/render.json
+  - Route/preview/1.0/route.json
+  - Search/stable/2023-06-01/search.json
+  - Spatial/stable/2022-08-01/spatial.json
+  - Timezone/preview/1.0/timezone.json
+  - Traffic/preview/1.0/traffic.json
+  - Weather/stable/1.1/weather.json
+```
```

Comment:
I am not sure whether service wish to add a "stable" tag for multiple preview Swaggers.

Reply:
@weidongxu-microsoft These changes were already approved and checked in. See [here](https://github.com/stevemunk/azure-rest-api-specs/commit/1d5886bb0ffec4abcf7cea5899aa77a8eeed8a89) for the commit.

Reply:
OK. If this already in, I am fine with it.

Just confused why it shows up in the diff.

Reply:
I'm also a bit confused that it's showing up on the diff.